test_util/aws.py

```diff
@@ -64,9 +64,12 @@ def create_stack(self, name, template_url, user_parameters, deploy_timeout=60):
 
 
 class CfStack():
-    def __init__(self, stack_name, boto_wrapper):
+    def __init__(self, stack_name, boto_wrapper, ssh_user=None, ssh_home_dir=None):
```

Comment:
"ssh access" stuff seems like it doesn't really belong in the CfStack class / seems like a separate concern.

It also doesn't make any sense for things like the SingleMaster AWS templates to set these on the CfStack.